### DISCUSSION TITLE
Inserting tab character in docs

### DISCUSSION THREAD

**Discussion by Snuupy** [outline/outline#2189] [Self Hosting]
Is there a way to insert the tab character in notes like in a standard text editor? I often use tabs to group together items or ideas without wanting to use bullet points.

**Comment by tommoor** [comment]
I'm afraid the underlying editor framework does not include support for tab characters. If it did, including this functionality would also make it difficult to navigate out of the editor using standard keyboard accessibility commands as tab is used to navigate interactive elements on the page.

If you have a screenshot of the type of information grouping you're trying to achieve it might help to think through how else it could be built out.

**Reply by Snuupy** [reply]
There's no reason I can't use bullet points instead, I'm just used to hitting the tab button for grouping information.

As a follow up, is there a way to mix in to do lists and bullet points? I notice that I can paste in the markdown like so, but I can't create the same data/structure using the built in editor (`Ctrl+]` doesn't work on the indented tasks).

- title1
  - [ ] task1
  - [ ] task2
- title2
  - [ ] task3

This is useful for grouping tasks together without forcing the titles of the tasks to be tasks themselves.

**Reply by tommoor** [reply]
Can definitely see how this is useful, you've found a good workaround with the markdown paste – there really isn't a way to achieve this in the editor right now because it doesn't like the mixing of list item types. This would be a good improvement

**Reply by spacec0de** [reply]
A solution to this would be really helpful as we use a lot of mixed lists as well. 

Lack of support for mixed-lists and tab characters are actually the biggest sources of pushback I get for user adoption. I guess non-coders are just really used to MS Word formatting options.
